webrtc/webrtc.php

```diff
@@ -42,7 +42,7 @@ function webrtc_content(&$a) {
 
         /* embedd the landing page in an iframe */
         $o .= '<h2>'.DI::l10n()->t('Video Chat').'</h2>';
-        $o .= '<p>'.DI::l10n()->t('WebRTC is a video and audio conferencing tool that works with Firefox (version 21 and above) and Chrome/Chromium (version 25 and above). Just create a new chat room and send the link to someone you want to chat with.').'</p>';
+        $o .= '<p>'.DI::l10n()->t('WebRTC is a video and audio conferencing tool that works in all modern browsers. Just create a new chat room and send the link to someone you want to chat with.').'</p>';
```

Comment:
The `C/messages.po` file needs to be recreated... I'll hopefully make a pull request for it later.

Reply:
oh sorry, I just assumed that happened automagically :)

Reply:
It's something we've been considering, but letting the merge request checks change the content of a branch is something we aren't comfortable with just yet.